### ISSUE TITLE
Don't run migrations in "testing" environment

### ISSUE THREAD

**Issue by kfirba** [eyewitness/eye#63]
Hey.

I think it would be wise not to run eyewitness migrations if the environment is "testing" (default Laravel's testing environment) or yet even better, make it configurable to decide on which environments to disable eyewitness.

Alternatively, maybe you can expose an "enabled" config option which we can set to "false" in "phpunit.xml" file.

What do you think?

**Comment by laurencei** [comment]
Hi @kfirba - yeah, great idea.

Leave it with me... I've got some ideas...

**Comment by arukompas** [comment]
Is there any drawback to running those migrations in the local environment? It's just a few extra tables, don't see any harm really.

**Comment by kfirba** [comment]
Not really, you can safely run them on “local” environment. You actually
*should* run them on “local” environment to ensure you get the result you
expect before deploying.
